src/main/resources/application.properties

```diff
@@ -33,4 +33,6 @@ auth.resourceId=fulfillment
 referencedata.url=${BASE_URL}
 notification.url=${BASE_URL}
 
-email.noreply=${MAIL_ADDRESS}
\ No newline at end of file
+email.noreply=${MAIL_ADDRESS}
+
+order.export.includeZeroQuantity=${ORDER_EXPORT_INCLUDE_ZERO_QUANTITY:false}
```

Comment:
Can you update readme with description for that env variable? Or just submit PR for openlmis-config.